src/connector/src/source/base.rs

```diff
@@ -543,6 +543,19 @@ pub struct SourceMessage {
     pub meta: SourceMeta,
 }
 
+impl SourceMessage {
+    /// Create a dummy `SourceMessage` with all fields unset for testing purposes.
+    pub fn dummy() -> Self {
```

Comment:
Is the `SourceContext::dummy` still in use?

Reply:
Yes.